.gitignore

```diff
@@ -3,3 +3,4 @@
 env
 *.xpr
 
+*.xpr
```

Comment:
this might have been left over from a merge conflict - it probably isn't hurting anything, but maybe it's worth cleaning up.